src/patches/modifierkey.cc

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include "config.h"
+#include "utils.h"
+#include <abseil-cpp/absl/strings/str_split.h>
+#include <iostream>
+#include <key.h>
+#include <modifierkey.h>
+#include <string>
+#include <string_view>
+
+ModifierKey::ModifierKey() {}
+
+ModifierKey* ModifierKey::Parse(std::string_view key)
+{
+  auto strippedKey = absl::StripTrailingAsciiWhitespace(key);
+  auto upperKey    = absl::AsciiStrToUpper(strippedKey);
+  auto wantedKeys  = absl::StrSplit(upperKey, "-", absl::SkipWhitespace());
+
+  auto modifierKey = new ModifierKey();
```

Comment:
Same as the other thing. I fail to see a good reason for this to be a heap allocated object per modified key that is just a pointer that will never get freed